desktop/ui/src/main/resources/datacleaner-home/jobs/Address cleansing with EasyDQ.analysis.xml

```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<job xmlns="http://eobjects.org/analyzerbeans/job/1.0">
-    <job-metadata>
-        <job-description>Corrects and validates addresses (credentials reqd)</job-description>
```

Comment:
This part is deleted, so doesn't matter ... but "reqd" is a pretty common (I think) shorthand for "required"